### ISSUE TITLE
computed is not defined

### ISSUE THREAD

**Issue by nilswinking** [Intevel/nuxt-directus#152]
### Version

nuxt-directus v5.4.0
nuxt: v3.5.3

### Steps to reproduce

1. Create a new nuxt project
2. Install directus-nuxt@5.4.0
3. Add directus-nuxt to the modules array in the nuxt config
4. Start nuxt

### What is Expected?

Nuxt and directus-nuxt start without an issue and no error is thrown

### What is actually happening?

```
[nuxt] [request error] [unhandled] [500] computed is not defined                                                                                                                                                                17:25:45
  at Module.useDirectusToken (C:\Users\xxx\directus-nuxt-test\node_modules\nuxt-directus\dist\runtime\composables\useDirectusToken.mjs:54:28)
  at Module.useDirectus (C:\Users\xxx\directus-nuxt-test\node_modules\nuxt-directus\dist\runtime\composables\useDirectus.mjs:10:105)
  at Module.useDirectusAuth (C:\Users\xxx\directus-nuxt-test\node_modules\nuxt-directus\dist\runtime\composables\useDirectusAuth.mjs:11:42)
  at Object.setup (C:\Users\xxx\directus-nuxt-test\node_modules\nuxt-directus\dist\runtime\plugin.mjs:11:47)
  at Object.wrapper (C:\Users\xxx\directus-nuxt-test\node_modules\nuxt\dist\app\nuxt.js:191:21)
  at C:\Users\xxx\directus-nuxt-test\virtual:nuxt:C:\Users\xxx\directus-nuxt-test\.nuxt\plugins\server.mjs:23:87
  at Module.executeAsync (/C:/Users/xxx/directus-nuxt-test/node_modules/unctx/dist/index.mjs:111:19)
  at Object.setup (C:\Users\xxx\directus-nuxt-test\virtual:nuxt:C:\Users\xxx\directus-nuxt-test\.nuxt\plugins\server.mjs:23:63)
  at wrapper (C:\Users\xxx\directus-nuxt-test\node_modules\nuxt\dist\app\nuxt.js:191:21)
  at C:\Users\xxx\directus-nuxt-test\node_modules\nuxt\dist\app\nuxt.js:111:58
```

**Comment by Intevel** [comment]
Fixed with https://github.com/Intevel/nuxt-directus/commit/65c1cdfaaabbf84d74fbb3ab68f492949f657ea9

**Comment by Intevel** [comment]
Release today in the evening.

**Comment by notflip** [comment]
Hi @Intevel Not to rush you! But do you have any idea when this will be released? I tried using the commit hash in my package.json to already test with this fix, but it didn't work.

**Comment by Intevel** [comment]
> Hi @Intevel Not to rush you! But do you have any idea when this will be released? I tried using the commit hash in my package.json to already test with this fix, but it didn't work.

Give me 40 minutes.

**Comment by Intevel** [comment]
> > Hi @Intevel Not to rush you! But do you have any idea when this will be released? I tried using the commit hash in my package.json to already test with this fix, but it didn't work.
> 
> Give me 40 minutes.

Im actually trying to publish it currently, but have no access. I think npm has some problems currently, I reauthed and switched devices. We have to wait:
![image](https://github.com/Intevel/nuxt-directus/assets/59223342/792c88a5-c3b5-4fda-b0ce-89e57b44bc0f)

**Comment by notflip** [comment]
> 

No worries at all! Thanks for this package.

**Comment by Intevel** [comment]
Fixed in v5.4.1
